.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!-- If writing isn't your strength, ask Janek for help :-) -->
```

Comment:
This is a public repo - we need to direct people to discord.

Reply:
Is this the right link? https://discord.com/invite/ngXmwvjSYF

Reply:
Added.